hibernate-reactive-core/src/test/java/org/hibernate/reactive/MutinySessionTest.java

```diff
@@ -56,6 +56,32 @@ private static String nameFromResult(List<Object> rowSet) {
 		}
 	}
 
+	@Test
+	public void testClearSessionAndChangeName(TestContext context) {
+		final GuineaPig pig1 = new GuineaPig( 81, "pig1" );
+		final GuineaPig pig2 = new GuineaPig( 82, "pig2" );
```

Comment:
Why are you creating two entities. I think the original issue was about:

1. session.persist pig1
2. session.clear
3. session.persist pig1
4. session.flush

This steps, without the fix, would cause a Persistence exception because it looks like we are trying to insert the same entity twice. But now you are trying to do something different that I don't follow.

Reply:
I see that the original user is actually doing an update of the entity. But can we simplify it and use a single entity instead?

Reply:
yes, probably

Reply:
re-worked the tests